Stack/Opc.Ua.Core/Security/Certificates/IPemResolver.cs

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 1996-2022 The OPC Foundation. All rights reserved.
+   The source code in this file is covered under a dual-license scenario:
+     - RCL: for OPC Foundation Corporate Members in good-standing
+     - GPL V2: everybody else
+   RCL license terms accompanied with this source code. See http://opcfoundation.org/License/RCL/1.00/
+   GNU General Public License as published by the Free Software Foundation;
+   version 2 of the License are accompanied with this source code. See http://opcfoundation.org/License/GPLv2
+   This source code is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*/
+
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Opc.Ua.Security.Certificates
+{
+    /// <summary>
+    /// An abstract interface for the Pem Resolver.
+    /// </summary>
+    public interface IPemResolver
+    {
+        /// <summary>
+        /// If the PEM resolver is actively configured.
+        /// </summary>
+        bool Active { get; }
+
```

Comment:
added helper to determine if the Pemresolver is really configured

Reply:
Removed